src/languages/php.js

```diff
@@ -52,12 +52,12 @@ export default function(hljs) {
     contains: hljs.QUOTE_STRING_MODE.contains.concat(SUBST),
   });
   const HEREDOC = hljs.END_SAME_AS_BEGIN({
-    begin: /<<<[ \t]*(\w+)\n/,
+    begin: /<<<[ \t]*"?(\w+)"?\n/,
     end: /[ \t]*(\w+)\b/,
     contains: hljs.QUOTE_STRING_MODE.contains.concat(SUBST),
   });
   const NOWDOC = hljs.END_SAME_AS_BEGIN({
-    begin: /<<<[ \t']*(\w+)'?\n/,
+    begin: /<<<[ \t]*'(\w+)'\n/,
```

Comment:
It must always be quoted?

Reply:
Fixed!

Reply:
But the end of now doc doesn't have to be quoted?